src/xenia/gpu/gl4/command_processor.cc

```diff
@@ -335,7 +335,7 @@ bool CommandProcessor::SetupGL() {
       "    }\n"
       "    EmitVertex();\n"
       "    EndPrimitive();\n"
-      "  } else {\n"
+      "  } else if (gl_in[0].gl_Position.y == gl_in[1].gl_Position.y) {\n"
```

Comment:
what about the fall-through case? (if it's even possible?) this shader won't emit anything without the else